packages/web_drivers/lib/chrome_driver_installer.dart

```diff
@@ -100,6 +110,23 @@ class ChromeDriverInstaller {
     return int.parse(versionNo);
   }
 
+  /// Find Google Chrome App on Mac.
+  Future<String> findChromeExecutableOnMac() async {
+    final io.ProcessResult processResult = await io.Process.run('find',
+        <String>['/Applications', '-type', 'd', '-name', '*Chrome.app']);
+
+    if (processResult.exitCode != 0) {
+      throw Exception('Failed to locate system Chrome.');
+    }
+    // The output looks like: /Applications/Google Chrome.app
+    String output = processResult.stdout as String;
+    output = output.trim();
+
+    String chromeExecutable = '${output}\/Contents\/MacOS\/Google Chrome';
```

Comment:
Consider using `path.join`. I think it also handles the escaping.

Reply:
worked like a charm! thanks! :) removed the strings dependancy.